diff --git a/dist/index.js b/dist/index.js
index c7aa4e66374759fd689fa65f1afc22e76c22cfae..a90dd6fa54823dd15634a19f908208e422d7e82e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -574,8 +574,15 @@ function generateTransforms(config, factories, manualSearchParams, logger) {
     };
 }
 
+function getNormalSize({ width, height, orientation }) {
+  return (orientation || 0) >= 5
+    ? { width: height, height: width }
+    : { width, height };
+}
+
 async function applyTransforms(transforms, image, removeMetadata = true) {
     image[METADATA] = { ...(await image.metadata()) };
+    image[METADATA] = {...image[METADATA], ...getNormalSize(image[METADATA])};
     if (removeMetadata) {
         // delete the private metadata
         delete image[METADATA].exif;
